api/envoy/extensions/filter/udp/dns_filter/v3/dns_filter.proto

```diff
@@ -0,0 +1,100 @@
+syntax = "proto3";
```

Comment:
why is there a copy of the proto file in 2 locations?

Reply:
The proto file is migrated to the v3 api.  This is the result of running`tools/proto_format/proto_format.sh fix`.